test/e2e/basic_scheduling/basic_scheduling_test.go

```diff
@@ -70,6 +70,17 @@ var _ = ginkgo.Describe("", func() {
 		Ω(err3).NotTo(HaveOccurred())
 		Ω(d).NotTo(BeNil())
 
+		ginkgo.By("Restart scheduler pod")
+		var podList, err4 = kClient.GetPods(configmanager.YuniKornTestConfig.YkNamespace)
+		Ω(err4).NotTo(HaveOccurred())
+		Ω(podList).NotTo(BeNil())
+		_, err5 := kClient.ScaleDeployment(configmanager.YKScheduler, 0, configmanager.YuniKornTestConfig.YkNamespace)
+		gomega.Ω(err5).NotTo(gomega.HaveOccurred())
+		err6 := kClient.WaitForPodBySelectorTerminated(configmanager.YuniKornTestConfig.YkNamespace, fmt.Sprintf("component=%s", configmanager.SchedulerName), 10)
+		kClient.ScaleDeployment(configmanager.YKScheduler, 1, configmanager.YuniKornTestConfig.YkNamespace)
+		gomega.Ω(err6).NotTo(gomega.HaveOccurred())
+		kClient.WaitForPodBySelectorRunning(configmanager.YuniKornTestConfig.YkNamespace, fmt.Sprintf("component=%s", configmanager.SchedulerName), 10)
+
```

Comment:
@wilfred-s Apologies if I missed the mark here, but I couldn't find a better place to include scheduler pod restart than the `BeforeSuite` of `basic_scheduling_test`. Please do advise if you have a better place in mind.